src/server/pachyderm_test.go

```diff
@@ -4029,6 +4032,84 @@ func TestChainedPipelinesNoDelay(t *testing.T) {
 	require.Equal(t, 2, len(jobInfos))
 }
 
+// TestMetaAlias tracks https://github.com/pachyderm/pachyderm/pull/8116
+// DAG:
+//
+// A
+// |
+// B
+// |
+// C
+func TestMetaAlias(t *testing.T) {
```

Comment:
I think the name of this test should be a little more descriptive - maybe `TestStartInternalPipeline`? I think we should include some comments to describe that this was done in response to a problem where under some circumstances when a pipeline in the middle of a DAG is Started, the that Pipeline will get stuck in the future due to a missing Meta Commit, followed by a missing Output commit.